src/Contrib/Grpc/GrpcTransportFactory.php

```diff
@@ -80,6 +92,15 @@ public function create(
         );
     }
 
+    public static function method(string $signal): string
```

Comment:
Should be moved to `Otlp`.

Reply:
k, I'll also move the user-agent header over to Otlp, since that's only a requirement for otlp exporting